ci/ci-iwyu.py

```
@@ -670,15 +793,68 @@ def main() -> int:
+    # Resolve board aliases to the canonical fbuild env / PIO build dir name.
+    # Must match ci-cppcheck.py so both tools see the same backend signal.
+    try:
+        from ci.boards import create_board
+
+        canonical_board_name = create_board(args.board).board_name
+    except KeyboardInterrupt as ki:
+        from ci.util.global_interrupt_handler import handle_keyboard_interrupt
+
+        handle_keyboard_interrupt(ki)
+        raise
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)